### DISCUSSION TITLE
Aqara FP2 compatibility

### DISCUSSION THREAD

**Discussion by matatools** [Koenkk/zigbee2mqtt#19390] [Devices]
It seems is stil not supported. Anybody could pair with Zigbee2mqtt?

**Comment by shawwellpete** [comment]
Aqara FP2 is wifi based?

**Reply by matatools** [reply]
Oh my god! You are right. I have other zigbee sensors and thought it was zigbee as well. Sorry

**Reply by shawwellpete** [reply]
No problem! A zigbee version would be cool...
